packages/ui-components/src/utils/date.helper.ts

```diff
@@ -1,90 +1,104 @@
-import moment, { Moment, MomentInput, unitOfTime } from 'moment';
-import momentTimezone from 'moment-timezone';
-import { SelectedRange } from '../types';
+import dayjs from 'dayjs';
+import { SelectedRange, DateInput } from '../types';
+
+import advancedFormat from 'dayjs/plugin/advancedFormat';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+import localeData from 'dayjs/plugin/localeData';
+import objectSupport from 'dayjs/plugin/objectSupport';
+import timezone from 'dayjs/plugin/timezone';
+import utc from 'dayjs/plugin/utc';
+import { TIMEZONES } from './date.config';
+
+dayjs.extend(advancedFormat);
+dayjs.extend(customParseFormat);
+dayjs.extend(localeData);
+dayjs.extend(objectSupport);
+dayjs.extend(timezone);
+dayjs.extend(utc);
 
 /**
  * Helper methods for date type display, parse, calculate and manipulation.
  *
  * @note The month number should be a number between 1 and 12, where 1 corresponds to January and 12 December.
  */
 
+const newDate = (date: DateInput) => dayjs(date);
+
+// Constructors
+export const fromISO = (date: string): dayjs.Dayjs => newDate(date);
+export const fromDateFields = (day: number, month: number, year: number): dayjs.Dayjs => newDate({ day, month: month - 1, year });
+export const fromDateInput = (date: DateInput): dayjs.Dayjs => newDate(date);
+
 // Months
-export const getMonthName = (month: number): string => `${moment({ month: month - 1 }).format('MMMM')}`;
-export const getNumberOfDaysInMonth = (month: number, year: number): number => moment({ month: month - 1, year }).daysInMonth();
+export const getMonthName = (month: number): string =>
```

Comment:
can this be a one liner?

Reply:
Actually, it was my code formatter, as it has exactly 101 chars (1 after other projects limited :stuck_out_tongue_closed_eyes: ). I just checked, and this package does not have a `prettier:fix` or `format:fix` script. Should I open a PR adding this?

Other thing, the config as a limit of 180 chars, is that not too long? Should I make it one line since it is under that limit?

Reply:
we have a `lint:fix` on this pkg if that's what you're looking for, or am I missing something here?

Reply:
I wasn't aware that `lint:ts:fix` was also formatting the code, thank you :+1: I think it is not taking in account the `.prettierrc.json` about `printWidth` though :thinking:

Reply:
we have `eslint-plugin-prettier` and `eslint-config-prettier` installed as dependencies in this project and prettier seems to be reporting warnings and errors, however `printWidth` isn't working like you said, could it have something to do with this [bug](https://github.com/eslint/eslint/issues/11408)?

Reply:
I think https://github.com/prettier/prettier-eslint could be a better approach than relying on `eslint-plugin-prettier` to do all the formatting. The [README.md](https://github.com/prettier/eslint-plugin-prettier#readme) of `eslint-plugin-prettier` says:

> If your desired formatting does not match Prettier’s output, you should use a different tool such as [prettier-eslint](https://github.com/prettier/prettier-eslint) instead.

Reply:
That would run prettier formatting before running eslint, to me it looks like a better approach so if there are no other implications behind the change, we can replace it :D what do you think? @djcouto @miguelpinto98

Reply:
This is out of the scope of this PR. Please open an issue so this can be discussed by @kelvininc/frontend team 👍